pages/linux/certbot.md

```diff
@@ -0,0 +1,28 @@
+# certbot
+
+> The Let's Encrypt Agent for automatically obtaining and renewing TLS certificates.
+> Successor to `letsencrypt`.
+
+- Obtain a new certificate via webroot authorization:
+
+`sudo certbot certonly --webroot --webroot-path {{path/to/webroot}} --domain {{subdomain.example.com}}`
+
+- Obtain a new certificate via nginx authorization, installing the new certificate automatically:
+
+`sudo certbot --nginx --domain {{subdomain.example.com}}`
+
+- Obtain a new certificate via apache authorization, installing the new certificate automatically:
+
+`sudo certbot --apache --domain {{subdomain.example.com}}`
+
+- Renew all Let's Encrypt certificates that expire soon (don't forget to restart any servers that use them afterwards):
```

Comment:
What is the actual expiry threshold ?

Reply:
As far as I know it's 30 days. :thinking:

Reply:
Ok, we should just mention the exact days then. Much more helpful IMO.